gitlab/v4/cli.py

```diff
@@ -268,12 +267,11 @@ def extend_parser(parser):
     # populate argparse for all Gitlab Object
     classes = []
     for cls in gitlab.v4.objects.__dict__.values():
-        try:
-            if gitlab.base.RESTManager in inspect.getmro(cls):
-                if cls._obj_cls is not None:
-                    classes.append(cls._obj_cls)
-        except AttributeError:
-            pass
+        if not isinstance(cls, type):
+            continue
+        if issubclass(cls, gitlab.base.RESTManager):
+            if cls._obj_cls is not None:
+                classes.append(cls._obj_cls)
```

Comment:
Do you think this could be simplified even more? Or did I miss something?

```suggestion
        if not issubclass(cls, gitlab.base.RESTManager):
            continue
        if cls._obj_cls is not None:
            classes.append(cls._obj_cls)
```

Reply:
Need the isinstance() check or it blows up.

```
>>> issubclass("hello", object)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: issubclass() arg 1 must be a class
>>>
```

Reply:
Ah, makes sense then. Thanks!